src/golang/lib/models/shared/retention.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/aqueducthq/aqueduct/lib/models/utils"
 )
 
-// A RetentionPolicy specifies that only KLatestRuns should be saved for each
```

Comment:
Not sure why some of these files are marked as edited because I didn't change them in the eng-1905 & eng-1906 branches. You can probably ignore these.